src/index.js

```diff
@@ -48,20 +53,26 @@ class Datastore {
   }
 
   _create(data, params = {}) {
-    let entities,
+    let entities = [],
         key;
 
-    if (data.hasOwnProperty(this.id)) {
-      key = this.makeKey(data[this.id], params);
-    } else {
-      key = this.makeKey(undefined, params);
-    }
+    data = Array.isArray(data) ? data : [data];
 
-    entities = { key, data };
+    data.map(item => {
+      if (item.hasOwnProperty(this.id)) {
+        key = this.makeKey(item[this.id], params);
+      } else {
+        key = this.makeKey(undefined, params);
+      }
+      entities.push({
+        key,
+        data: item,
+      });
+    });
 
-    // Normalize
-    if (Array.isArray(data)) {
-      entities = data.map(data => ({ key, data }));
+    // unbox if only 1 entity is inserted
+    if (data.length === 1) {
+      entities = entities[0];
```

Comment:
Small thing - if this is called with an array of items, but only 1 item e.g.

```js
service.create([{
  id: 'foo',
  value: 'bar'
}]);
```

it'll respond with:

```js
{
  id: 'foo',
  value: 'bar'
}
```

rather than

```js
[{
  id: 'foo',
  value: 'bar'
}]
```

because it normalizes into an array, and then unboxes if length is 1. 

Should just be fixed by adding a flag before boxing up the data e.g. at the top:

```js
let wasArray = Array.isArray(data);

...

if (wasArray && data.length === 1) {
  entities = entities[0];
}
```